sys/dev/cxgbe/tom/t4_ddp.c

```diff
@@ -1259,7 +1259,9 @@ hold_aio(struct toepcb *toep, struct kaiocb *job, struct pageset **pps)
 {
 	struct vmspace *vm;
 	vm_map_t map;
-	vm_offset_t start, end, pgoff;
+	vm_offset_t pgoff;
+	char * __capability start;
+	char * __capability end;
```

Comment:
```suggestion
	char * __capability start, * __capability end;
	vm_offset_t pgoff;
```
(or `pgoff` can come first, I just figured it's nicer to keep the same order as upstream)